site/src/components/pagetitle.js

```diff
@@ -1,11 +1,19 @@
 import React from "react"
 import { css } from "@emotion/core"
-import { spacing } from "../utils/styles"
+import { spacing, colors } from "../utils/styles"
 import Metadata from "./metadata"
 import { getSiteSpecificStyle } from "../utils/contenttransforms"
 
-export default ({ children, icon = null, subtitle = null, pageType }) => {
+export default ({
+  children,
+  icon = null,
+  subtitle = null,
+  roles = "",
+  pageType,
+  isArticle = false
```

Comment:
isArticle? Why is this here? This is putting article-specific concerns into a general-purpose title component.  If there is a special kind of title required for an article page, there should be an articletitle class which extends the pagetitle class.  This is a bit back-to-front.

Reply:
refactored